src/commands/CleanCommand.js

```diff
@@ -1,16 +1,22 @@
 import async from "async";
 import Command from "../Command";
 import FileSystemUtilities from "../FileSystemUtilities";
+import PackageUtilities from "../PackageUtilities";
 import PromptUtilities from "../PromptUtilities";
 import progressBar from "../progressBar";
 
 export default class CleanCommand extends Command {
   initialize(callback) {
+    this.packagesToClean = this.filteredPackages;
+    if (this.flags.includeFilteredDependencies) {
+      this.packagesToClean = PackageUtilities.addDependencies(this.packagesToClean, this.packageGraph);
```

Comment:
I was about to ask why not just mutate `this.filteredPackages` in place, but then I realized this is a pattern used elsewhere and it makes it more readable to boot. 👓